### ISSUE TITLE
Feature Requested: Refactor softmax cross entropy benchmark

### ISSUE THREAD

**Issue by fufjvnvnf** [stevenygd/NN.jl#8]

**Comment by stevenygd** [comment]
#9
